harness/determined/cli/user.py

```diff
@@ -173,7 +209,8 @@ def whoami(parsed_args: Namespace) -> None:
         Cmd("rename", rename, "change username for user", [
             Arg("target_user", default=None, help="name of user whose username should be changed"),
             Arg("new_username", default=None, help="new username for target_user"),
-        ]),
+        ], is_default=False,
```

Comment:
I'm not sure what to make of the `is_default` addition when it's the default and not included in other `Cmd`s. What are you communicating?

Reply:
Sorry this was added as I believed that all optional arguments had to be filled if later ones were filled. Removed